pkg/controllers/status/cluster_status_controller.go

```diff
@@ -146,9 +136,20 @@ func (c *ClusterStatusController) syncClusterStatus(cluster *clusterv1alpha1.Clu
 		klog.V(2).Infof("Cluster(%s) still offline after retry, ensuring offline is set.", cluster.Name)
 		currentClusterStatus.Conditions = generateReadyCondition(false, false)
 		setTransitionTime(&cluster.Status, &currentClusterStatus)
+		c.InformerManager.Stop(cluster.Name)
```

Comment:
@mrlihanbo Hello Hanbo, how are you doing?

Can you remember the reason why stopping the informer here, in case of a cluster offline?

#2930 is now trying to solve an issue due to this change.

also, cc @Garrybest to help recall.

Reply:
Maybe because we want to re-establish the informer after the apiserver is healthy? 🤔

The reason may be not so convincible because I don't remember this line as well. 🤣

Reply:
I talked to @mrlihanbo, he said this is probably for disabling repetitive warning logs, especially for those clusters offline for a long time.